Assets/Game/Users/Grahame/OutlineDemo.unity

```diff
@@ -1,3 +1,125 @@
-version https://git-lfs.github.com/spec/v1
-oid sha256:ef7beb27a15b61c395d69c39cdb7c6c9c449d5c92757b3bb1f4bcd5fd823dcbb
-size 145784
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!29 &1
```

Comment:
You will probably get a merge conflict on this when pulling from develop. Accept incoming changes.